tests/testthat/test_pipeop_newtarget.R

```diff
@@ -0,0 +1,20 @@
+test_that("convert_task reconstructs task", {
```

Comment:
could you copy all relevant tests?  (Or possibly test a few more edge cases).

In theory, this function should also be able to just change iris into a regr task by predicting `Sepal.Length` instead of `Species` etc. guess we  can do a few more sensible tests.